src/NuGet.Packaging/PackagesConfigReader.cs

```diff
@@ -122,11 +123,21 @@ public NuGetVersion GetMinClientVersion()
         }
 
         /// <summary>
-        /// Reads all package node entries in the config
+        /// Reads all package node entries in the config.
+        /// If duplicate package ids exist an exception will be thrown.
         /// </summary>
-        /// <returns></returns>
         public IEnumerable<PackageReference> GetPackages()
         {
+            return GetPackages(allowDuplicatePackageIds: false);
```

Comment:
@yishaigalatzer, @emgarten - should this be set to true given the response below?